eagleproject/core/blockchain/harvest/snapshots.py

```diff
@@ -683,6 +684,7 @@ def latest_snapshot_block_number(project=OriginTokens.OUSD):
 
 
 def calculate_snapshot_data(block=None):
+    project = OriginTokens.OUSD
```

Comment:
if this function is for OUSD only, should we change the name to `calculate_snapshot_data_ousd`?

Reply:
Yeah, will rename it